src/globus_cli/parsing/param_types/location.py

```diff
@@ -18,20 +18,27 @@ class LocationType(AnnotatedParamType):
     def get_type_annotation(self, param: click.Parameter) -> type:
         # mypy does not recognize this as a valid usage at runtime
         # ignore for now
-        return tuple[float, float]  # type: ignore[no-any-return,misc]
+        return str
 
     def convert(
         self, value: t.Any, param: click.Parameter | None, ctx: click.Context | None
     ) -> t.Any:
+        match_result = re.match(r"^([^,]+),([^,]+)$", value)
+        if not match_result:
+            self.fail(
+                f"location '{value}' does not match the expected "
+                "'latitude,longitude' format"
+            )
+
+        maybe_lat = match_result.group(1)
+        maybe_lon = match_result.group(2)
```

Comment:
I've never seen `maybe_x` variable naming convention and I genuinely can't tell if I love it or hate it.

Reply:
I still remember learning Haskell, which calls optional types "Maybe". `Maybe a` is what python writes as `Optional[a]`.
Somehow, that word, and the way that it adds uncertainty, has stuck with me as something I use from time to time.

Not really an explanation or even a defense of the naming! Just a bit of color. 🎨